CONTRIBUTING.md

```diff
@@ -18,30 +18,34 @@
 <summary><b>theme example</b></summary>
 
 To create a custom theme you need to define a colorscheme for each of vim's modes. Each mode has a `fg` and `bg` field for every lualine section.
+You can add special effects with `sp`. 
```

Comment:
I would personally change `sp` to `gui` since this is what is used to create vim's `highlight` commands.

Reply:
ok I'll change that

Reply:
both cterm and gui create those effects .  
Thats why I thought of not going with gui.

Reply:
Yea the naming convention for highlights is weird but I think it's better to stick with it to avoid further confusion